tests/Unit/PackagePrivate/Humanizer/Internationalization/FallbackMessageBuilderTest.php

```diff
@@ -14,27 +14,59 @@
  */
 class FallbackMessageBuilderTest extends TestCase
 {
-    private ArrayMessageBuilder $localeMessageBuilder;
-
-    protected function setUp(): void
+    public function testBuildMessageThrowsException(): void
     {
-        parent::setUp();
-        $this->localeMessageBuilder = $this->createMock(ArrayMessageBuilder::class);
-        $this->localeMessageBuilder
+        $primaryBuilder = $this->createMock(ArrayMessageBuilder::class);
+        $primaryBuilder
             ->method('buildMessage')
             ->willThrowException(new UnknownMessageKey());
-    }
 
-    public function testBuildMessageThrowsException(): void
-    {
-        $fallbackMessageBuilder = new FallbackMessageBuilder($this->localeMessageBuilder, []);
+        $fallbackBuilder = $this->createMock(ArrayMessageBuilder::class);
+        $fallbackBuilder
+            ->expects($this->once())
+            ->method('buildMessage')
+            ->with('not-existing-key')
+            ->willThrowException(new UnknownMessageKey());
```

Comment:
Since ArrayMessageBuilder is easy to construct and has no side effects, you could just create two real instances. That avoids using the PHPUnit mock API.

Reply:
very nice point. I have changed this